### PR TITLE
Added show reservation date option

### LINKED ISSUE #8: Option to show the reservation date?
From @JulienPalard

### PR DESCRIPTION
**Pull request by Seluj78**
Closes #8

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,9 +27,10 @@ optional arguments:
   -f, --fuzzy           print only files marked as fuzzys
   -o, --offline         don't perform any fetching to GitHub/online
   -n, --no-reserved     don't print info about reserved files
-  -c, --counts          render list with the count of remaining entries
-                        (translate or review) rather than percentage done
+  -c, --counts          render list with the count of remaining entries (translate or review) rather than percentage done
   -j, --json            format output as JSON.
+  --show-reservation-dates
+                        Will show reservation_dates
```
Comment (christopheNan) [inline README.md:33]:
```suggestion
                        show issue creation dates 
```

File: potodo/_github.py [hunk 2]
```diff
@@ -67,6 +71,9 @@ def get_reservation_list(repo_path: Path) -> Mapping[str, str]:
         # Maybe find a better way for not using python 3.8 ?
         yes = re.search(r"\w*/\w*\.po", issue["title"])
         if yes:
-            reservations[yes.group()] = issue["user"]["login"]
+            datetime_object = datetime.strptime(
+                issue["created_at"].split("T")[0], "%Y-%m-%d"
+            ).date()
+            reservations[yes.group()] = (issue["user"]["login"], datetime_object)
```
Comment (christopheNan) [inline potodo/_github.py:77]:
```suggestion
            creation_date = datetime.strptime(
                issue["created_at"].split("T")[0], "%Y-%m-%d"
            ).date()
            reservations[yes.group()] = (issue["user"]["login"], creation_date)
```

File: potodo/potodo.py [hunk 3]
```diff
@@ -314,6 +327,13 @@ def main() -> None:
         help="format output as JSON.",
     )
 
+    parser.add_argument(
+        "--show-reservation-dates",
+        action="store_true",
+        dest="show_reservation_dates",
+        help="Will show reservation_dates",
```
Comment (christopheNan) [inline potodo/potodo.py:334]:
```suggestion
        help="show issue creation dates",
```

**Review by christopheNan — APPROVED** [review]
👍 ❤️

**Comment by Seluj78** [comment]
@christopheNan Can you review again ? It got dismissed

**Review by christopheNan — DISMISSED** [review]

**Pull request merged**